x-pack/platform/plugins/shared/alerting/server/types.ts

```diff
@@ -226,6 +226,7 @@ export const DEFAULT_AAD_CONFIG: IRuleTypeAlerts<DefaultAlert> = {
   context: `default`,
   mappings: { fieldMap: {} },
   shouldWrite: true,
+  dangerouslyCreateAlertsInAllSpaces: true,
```

Comment:
Should the default be false?

```suggestion
  dangerouslyCreateAlertsInAllSpaces: false,
```

Reply:
oo yea i did that for testing. good catch.